routes.go

```diff
@@ -143,6 +144,7 @@ func InitRoutes(apper Apper, r *mux.Router) *mux.Router {
 	write.HandleFunc("/admin", handler.Admin(handleViewAdminDash)).Methods("GET")
 	write.HandleFunc("/admin/users", handler.Admin(handleViewAdminUsers)).Methods("GET")
 	write.HandleFunc("/admin/user/{username}", handler.Admin(handleViewAdminUser)).Methods("GET")
+	write.HandleFunc("/admin/user/{username}", handler.Admin(handleAdminToggleUserSuspended)).Methods("POST")
```

Comment:
What do you think about making this `/admin/user/{username}/suspend` instead? As we might want to use `/admin/user/{username}` for other functions in the future.

Reply:
I went with `/admin/user/{username}/status` in aniticpation of possible future 'statuses' being added, still just toggles the status for now